docs/source/developer/documentation.rst

```diff
@@ -89,4 +89,13 @@ To do so, use the following command:
 
 .. code-block::
 
-    $ make -f docs/Makefile.sphinx html
\ No newline at end of file
+    $ make -f docs/Makefile.sphinx html
```

Comment:
we should probably just replace this line with `tox -e docs`

Reply:
Oh that is the recommended approach right above this code block.  This one says something like "And if for some reason you don't want to use tox, then you can do this:"  :)